rush-plugins/rush-azure-storage-build-cache-plugin/src/AdoCodespacesAuthCredential.ts

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+import type { AccessToken, GetTokenOptions } from '@azure/identity';
+
+export class AdoCodespacesAuthCredential {
+  // eslint-disable-next-line @rushstack/no-new-null
+  public async getToken(scopes: string | string[], options?: GetTokenOptions): Promise<AccessToken | null> {
+    if (!Array.isArray(scopes)) {
+      scopes = [scopes];
+    }
+    const { resolve } = await import('path');
+    const homeDir: string | undefined = process.env.HOME;
+    if (!homeDir) {
+      throw new Error('Could not determine the home directory');
+    }
+    const azureAuthHelperExec: string = resolve(homeDir, 'azure-auth-helper');
```

Comment:
```suggestion
    const homeDir: string | undefined = process.env.HOME;
    if (!homeDir) {
      throw new Error('Could not determine the home directory');
    }
    const azureAuthHelperExec: string = `${homeDir}/azure-auth-helper`;
```

Reply:
Is `azure-auth-helper` always there? Is it not always on the PATH?

Reply:
I can use it from PATH. The exec gets symlinked to `/usr/local/bin` when the extension boots
```
lrwxrwxrwx 1 root root 30 Apr 18 05:03 /usr/local/bin/azure-auth-helper -> /home/vscode/azure-auth-helper
```